compiler/rustc_parse_format/src/lib.rs

```diff
@@ -809,94 +838,97 @@ impl<'a> Parser<'a> {
 /// Finds the indices of all characters that have been processed and differ between the actual
 /// written code (code snippet) and the `InternedString` that gets processed in the `Parser`
 /// in order to properly synthesise the intra-string `Span`s for error diagnostics.
-fn find_skips_from_snippet(
+fn find_width_map_from_snippet(
     snippet: Option<string::String>,
     str_style: Option<usize>,
-) -> (Vec<usize>, bool) {
+) -> (Vec<InnerWidthMapping>, bool) {
     let snippet = match snippet {
         Some(ref s) if s.starts_with('"') || s.starts_with("r\"") || s.starts_with("r#") => s,
         _ => return (vec![], false),
     };
 
-    fn find_skips(snippet: &str, is_raw: bool) -> Vec<usize> {
+    fn find_width_map(snippet: &str, is_raw: bool) -> Vec<InnerWidthMapping> {
```

Comment:
This will be changed by https://github.com/rust-lang/rust/pull/104193.
cc @TaKO8Ki

Reply:
Given that there are conflicts from both #103651 and #104193, I'll go ahead and resolve the other issues before I try sort those out.